framework/src/task_manager/task_manager_core.c

```diff
@@ -67,10 +67,9 @@ static int task_manager_pid;
 #define TYPE_CANCEL      1
 #define TYPE_EXIT        2
 
-#define CB_FUNC_OF(X)             ((tm_termination_info_t *)X)->cb
-#define CB_DATA_OF(X)             ((tm_termination_info_t *)X)->cb_data
-#define CB_MSG_OF(X)              ((tm_msg_t *)CB_DATA_OF(X))->msg
-#define CB_MSGSIZE_OF(X)          ((tm_msg_t *)CB_DATA_OF(X))->msg_size
+#define CB_MSG_OF(X)        (X)->cb_data->msg
```

Comment:
Why is 
(tm_msg_t *) 
typecast not required anymore ?

Reply:
Previously, some values were passed as void type rather than tm_msg_t type. 
Therefore casting to tm_msg_t type was required. 
However, at #2443 commit and this commit these processes were replaced.
As a result, (tm_msg_t *) typecast is not requred anymore.